clients/cpp/src/signalrlib/HttpConnection.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#include <cpprest\http_client.h>
+#include <cpprest\json.h>
+#include <cpprest\ws_client.h>
+#include "HttpConnection.h"
+
+namespace signalr
+{
+    HttpConnection::HttpConnection(const utility::string_t& url)
+    {
+        web::http::client::http_client client(url);
+
+        web::http::http_request request;
+        request.set_method(L"POST");
```

Comment:
think cross plat from the very beginning or you will have a lot of work later. Start from using `_XPLATSTR` or `_U` for string literals